gemma-core/src/main/java/ubic/gemma/persistence/service/common/description/CharacteristicDao.java

```diff
@@ -162,4 +166,9 @@ class CharacteristicUsageFrequency {
      * the caller.
      */
     Map<Characteristic, Long> getParentIds( Class<?> parentClass, @Nullable Collection<Characteristic> characteristics );
+
+    /**
+     * Create a statement.
+     */
+    Statement create( Statement s );
```

Comment:
This shouldn't be necessary, statements should always be created in cascade from the enclosing FactorValue.